sdk/src/main/scala/io/horizen/account/api/http/route/AccountTransactionApiRoute.scala

```diff
@@ -79,7 +79,7 @@ case class AccountTransactionApiRoute(override val settings: RESTApiSettings,
       allForgingStakes ~ myForgingStakes ~ decodeTransactionBytes ~ openForgerList ~ allowedForgerList ~ createKeyRotationTransaction ~
       invokeProxyCall ~ invokeProxyStaticCall  ~ sendKeysOwnership ~ getKeysOwnership ~ removeKeysOwnership ~
       getKeysOwnerScAddresses ~ sendMultisigKeysOwnership ~ pagedForgingStakes ~ pagedForgersStakesByForger ~
-      pagedForgersStakesByDelegator ~ registerForger
+      pagedForgersStakesByDelegator ~ registerForger ~ updateForger
```

Comment:
Both `registerForger` and `updateForger` should be disabled in case of a seeder node. They should be added in `listOfDisabledEndpoints` method.

Reply:
Done